general/company_events.md

```diff
@@ -7,7 +7,7 @@ Once a month we organise a get together where we either all go to a bar where fr
 The source{d} summer party is hosted once a year in the months of June, July or September (depending on travel schedules). The reason for this party to exist is just celebrating how awesome summer is and because it's a good excuse to have some fun together. Everyone in the company and their +1s are invited to join. We usually book a place where snacks will be served as well as free drinks. Sometimes we organize some group activities before the party starts, so the starting time varies. The party lasts until the last person standing can't take it anymore and leaves. After such a tough night no one will expect you to show up at the office the next day, so it's fine to stay at home and recover.
 
 ## Christmas Party
-The source{d} Christmas party is hosted once a year in December. It's basically the winter version of the summer party where we gather to celebrate the end of another awesome year. Everyone in the company and their +1s are invited to join. We usually book a place where snacks will be served as well as free drinks. Sometimes we organize some group activities before the party starts, so the starting time varies. A few week beore the party we start a Secret Santa and the give out the presents during the party. The party lasts until the last person standing can't take it anymore and leaves. After such a tough night no one will expect you to show up at the office the next day, so it's fine to stay at home and recover.
+The source{d} Christmas party is hosted once a year in December. It's basically the winter version of the summer party where we gather to celebrate the end of another awesome year. Everyone in the company and their +1s are invited to join. We usually book a place where snacks will be served as well as free drinks. Sometimes we organize some group activities before the party starts, so the starting time varies. A few week before the party we start a Secret Santa and the give out the presents during the party. The party lasts until the last person standing can't take it anymore and leaves. After such a tough night no one will expect you to show up at the office the next day, so it's fine to stay at home and recover.
```

Comment:
`A few week before the party we start a Secret Santa and the give out the presents during the party.`?